code/game/gamemodes/cult/runes.dm

```diff
@@ -617,17 +638,23 @@ structure_check() searches for nearby cultist structures required for the invoca
 		fail_invoke()
 		return
 
-	..()
-	if(mob_to_revive.stat == DEAD)
-		var/diff = length(GLOB.sacrificed) - SOULS_TO_REVIVE - sacrifices_used
-		if(diff < 0)
-			to_chat(user, "<span class='cult'>Your cult must carry out [abs(diff)] more sacrifice\s before it can revive another cultist!</span>")
+	if(mob_to_revive.stat != DEAD && length(invokers) < 2)
```

Comment:
Can you change `req_cultists` on the rune variables instead? Perhaps altering `/obj/effect/rune/attack_hand` to include something like "This runes requires at least [req_cultists] acolytes to activate."

Reply:
If i change that, it will be impossible to revive dead cultist alone
My change supposed to keep reviving dead with one cultist invoking and healing living cultist with two invokers so no

Reply:
Ahh, alright. I see now. Thanks for the quick response!